*** lighttpd1.4/src/mod_cgi.c	Sun Sep  9 06:48:45 2018
--- mod_cgi.c	Sun Sep  9 06:53:19 2018
***************
*** 801,807 ****
  
  		cgi_pid_add(p, hctx->pid, hctx);
  
! 		if (0 == con->request.content_length) {
  			close(to_cgi_fds[1]);
  		} else {
  			/* there is content to send */
--- 801,807 ----
  
  		cgi_pid_add(p, hctx->pid, hctx);
  
! 		if (0 == con->request.content_length && !hctx->conf.upgrade) {
  			close(to_cgi_fds[1]);
  		} else {
  			/* there is content to send */
