*** /etc/lighttpd/modules.conf	Sun Sep  9 07:13:00 2018
--- modules.conf	Sun Sep  9 07:12:39 2018
***************
*** 45,53 ****
  #  "mod_auth",
  #  "mod_authn_file",
  #  "mod_evasive",
! #  "mod_redirect",
! #  "mod_rewrite",
! #  "mod_setenv",
  #  "mod_usertrack",
  )
  
--- 45,53 ----
  #  "mod_auth",
  #  "mod_authn_file",
  #  "mod_evasive",
!   "mod_redirect",
!   "mod_rewrite",
!   "mod_setenv",
  #  "mod_usertrack",
  )
  
***************
*** 92,98 ****
  ##
  ## mod_magnet
  ##
! #include "conf.d/magnet.conf"
  
  ##
  ## mod_cml
--- 92,98 ----
  ##
  ## mod_magnet
  ##
! include "conf.d/magnet.conf"
  
  ##
  ## mod_cml
***************
*** 107,113 ****
  ##
  ## mod_proxy
  ##
! #include "conf.d/proxy.conf"
  
  ##
  ## mod_expire
--- 107,113 ----
  ##
  ## mod_proxy
  ##
! include "conf.d/proxy.conf"
  
  ##
  ## mod_expire
***************
*** 135,146 ****
  ##
  ## FastCGI (mod_fastcgi)
  ##
! #include "conf.d/fastcgi.conf"
  
  ##
  ## plain old CGI (mod_cgi)
  ##
! #include "conf.d/cgi.conf"
  
  ##
  #######################################################################
--- 135,146 ----
  ##
  ## FastCGI (mod_fastcgi)
  ##
! include "conf.d/fastcgi.conf"
  
  ##
  ## plain old CGI (mod_cgi)
  ##
! include "conf.d/cgi.conf"
  
  ##
  #######################################################################
